app/src/test/java/com/example/concordia_campus_guide/ViewModel/MainActivityViewModelTest.java

```diff
@@ -0,0 +1,44 @@
+package com.example.concordia_campus_guide.ViewModel;
+
+import com.example.concordia_campus_guide.Activities.MainActivityViewModel;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockitoAnnotations;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class MainActivityViewModelTest {
+    private MainActivityViewModel mainActivityViewModel;
+
+    @Before
+    public void init(){
+        MockitoAnnotations.initMocks(this);
```

Comment:
Why do you need this?

Reply:
that's true it can be removed if you aren't using any mocks.

Reply:
True, I had there at the beginning thinking i will need to mock something. Good catch guys